installation/noobs.md

```diff
@@ -20,7 +20,8 @@ Once you've downloaded the NOOBS zip file, you'll need to copy the contents to a
 
 To set up a blank SD card with NOOBS:
 
-- Format an SD card which is 8GB or larger as FAT. See the instructions given below.
+- Format an SD card as FAT. See the instructions given below.
+  - Your SD card will need to be at least 16GB for Full Raspbian, or at least 8GB for all other installs. 
```

Comment:
I don't think "Full Raspbian" is ever mentioned as a term anywhere else? :man_shrugging: 
https://www.raspberrypi.org/downloads/raspbian/ says "Raspbian Buster with desktop and recommended software" but that's obviously a bit of a mouthful :wink: 

ping @XECDesign and @spl237

Reply:
According to os_list, at least in noobs, "Raspbian Full" seems to be used, with a longer description of "A port of Debian with desktop and recommended applications".

Reply:
Perhaps Full on the Noobs poage, but the long text on the SD card page...

Reply:
Which is sort of what I have done in the latest edit.

Reply:
Instead of "Full Raspbian", should this be "Raspbian Full" as Serge mentions above?